Configuration/Settings.yaml

```diff
@@ -4,6 +4,6 @@ Neos:
       Array: 'TYPO3\Eel\Helper\ArrayHelper'
 
     metaDataMapping:
-      title: '${asset.Title || iptc.Title || exif.imageDescription}'
+      title: '${String.crop(asset.Title || iptc.Title || exif.imageDescription, 255)}'
       caption: '${asset.Caption || iptc.Description}'
-      tags: '${Array.concat(asset.tags, iptc.keywords)}'
\ No newline at end of file
+      tags: '${Array.concat(asset.tags || [], iptc.keywords || [])}'
```

Comment:
It seems that if i.e. iptc.keywords is an empty string, it results in an array with one empty element resulting in the tried creation of a tag with empty name. It fixed the problem in our real live case. But, nevermind, is already solved in your PR!